tests/test-dirs/locate/module-aliases.t/run.t

```diff
@@ -65,12 +65,11 @@ Jump to an aliased module `A|.f`:
   }
 
 Jump to the declaration of an aliased module `A|.f`.
-With the new shape implementation it is expected that we jump to the 
-declaration of the alias and not to the aliased module itself.
+The alias is traversed.
```

Comment:
Seeing this, it seems like we had _decided_ to have the "stop on alias" behavior. Perhaps (thanks to the move to dune) the whole community has switched to using module aliases extensively and the behavior is now more annoying than anything.
But I do wonder if we could not perhaps add a flag to keep the old behavior to stop on aliases for declarations... not sure whether it's worth it. I guess it will depend how ugly the change required to `locate.ml` are?

Reply:
I feel like the most often expected behavior is to traverse the aliases. Let's keep it that way for now, and if the need to add a configuration flag arises we will do it at that time.